indra/llwebrtc/llwebrtc.cpp

```diff
@@ -516,13 +515,23 @@ void LLWebRTCImpl::updateDevices()
         char name[webrtc::kAdmMaxDeviceNameSize];
         char guid[webrtc::kAdmMaxGuidSize];
         mTuningDeviceModule->PlayoutDeviceName(index, name, guid);
+
+#if WEBRTC_LINUX
```

Comment:
I suggest turning this either into a function ala
```
LLWebRTCVoiceDevice createWebRTCVoiceDevice( std::string name, std::string guid )
{
#if LL_LINUX
    if( guid.empty() )
        guid = name;
#endif

    return {name, guid };
```

and then call this in the two places. Or just be lazy and modify the ctor of LLWebRTCVoiceDevice

```
    LLWebRTCVoiceDevice(const std::string &display_name, const std::string &id) :
        mDisplayName(display_name),
        mID(id)
    {
        if( mID.empty() )
            mID = mDisplayName;
    };
```

in any case is my suggestion to get rid of:
- the strcpy which can easily lead to buffer overflows if one of the enums above changes
- the duplicate code of checking for an empty GUID

Reply:
Done (the second option)

Reply:
Thank you :)